spec/models/navigation_rule_spec.rb

```diff
@@ -139,4 +139,25 @@
       expect(resource.smartanswer?).to be false
     end
   end
+
+  describe '#display_text' do
+    let(:resource) {
```

Comment:
Not a blocker: the `step_by_step_page` defined at the top of the test could be reused here rather than creating new object. 

e.g. `expect(step_by_step_page.display_text("always")).to eq("Always show navigation")`

Reply:
But I'm creating a `navigation_rule` here and I'm reusing the `step_by_step_page` created at the top. I must say that the name `resource` is not very descriptive but that's what is used in all the other tests.

Reply:
Sorry! I misread that. 🤦‍♂  You're right, `resource` is badly named.